include/lsst/meas/base/GaussianCentroid.h

```diff
@@ -127,6 +127,12 @@ class GaussianCentroidAlgorithm : public SimpleAlgorithm {
     SafeCentroidExtractor _centroidExtractor;
 };
 
+class GaussianCentroidTransform : public CentroidTransform {
+public:
+    typedef GaussianCentroidControl Control;
+    GaussianCentroidTransform(Control const & ctrl, std::string const & name, afw::table::SchemaMapper & mapper);
+};
```

Comment:
It's much more common in our codebase to have blank lines between each method, typedef, and member in a class (see standard 6-19, though reading that just now is the first time I've noticed that we actually suggest two blank lines in source files).